README.md

```diff
@@ -1,6 +1,6 @@
 # Orb Project Template
 <!---
-[![CircleCI Build Status](https://circleci.com/gh/<organization>/<project-name>.svg?style=shield "CircleCI Build Status")](https://circleci.com/gh/<organization>/<project-name>) [![CircleCI Orb Version](https://img.shields.io/badge/endpoint.svg?url=https://badges.circleci.io/orb/<namespace>/<orb-name>)](https://circleci.com/orbs/registry/orb/<namespace>/<orb-name>) [![GitHub License](https://img.shields.io/badge/license-MIT-lightgrey.svg)](https://raw.githubusercontent.com/<organization>/<project-name>/master/LICENSE) [![CircleCI Community](https://img.shields.io/badge/community-CircleCI%20Discuss-343434.svg)](https://discuss.circleci.com/c/ecosystem/orbs)
+[![CircleCI Build Status](https://circleci.com/gh/<organization>/<project-name>.svg?style=shield "CircleCI Build Status")](https://circleci.com/gh/<organization>/<project-name>) [![CircleCI Orb Version](https://badges.circleci.com/orbs/<namespace>/<orb-name>)](https://circleci.com/orbs/registry/orb/<namespace>/<orb-name>) [![GitHub License](https://img.shields.io/badge/license-MIT-lightgrey.svg)](https://raw.githubusercontent.com/<organization>/<project-name>/master/LICENSE) [![CircleCI Community](https://img.shields.io/badge/community-CircleCI%20Discuss-343434.svg)](https://discuss.circleci.com/c/ecosystem/orbs)
```

Comment:
@gmemstr @KyleTryon Is this endpoint supposed to work?
For example, with https://circleci.com/developer/orbs/orb/circleci/node the endpoint becomes https://badges.circleci.com/orbs/circleci/node but it's a 404. The same for every published orbs I tested.
Tried also https://badges.circleci.com/orb/circleci/node or https://badges.circleci.com/orbs/orb/circleci/node but all 404.
Sadly, when generating orb from the template it gives a broken image: 
![image](https://user-images.githubusercontent.com/175128/116972202-f76de100-acba-11eb-8cd2-d2a62b38d1ab.png)

Reply:
Thanks for #27!